BUILD

```diff
@@ -49,7 +49,53 @@ filegroup(
         "//testgrid/jenkins_verify:all-srcs",
         "//triage:all-srcs",
         "//velodrome:all-srcs",
-        "//vendor:all-srcs",
+        "//vendor/bitbucket.org/ww/goautoneg:all-srcs",
```

Comment:
what changed here? why isn't it just `"//vendor:all-srcs"`?

Reply:
I updated the PR body with the commands I used. This was changed by `verify/update-bazel.sh`.

Reply:
this is why we should make verify/ match k/k hack/ :-P